app/static/src/app/store/modelFit/actions.ts

```diff
@@ -64,6 +64,7 @@ export const actions: ActionTree<ModelFitState, FitState> = {
         commit(ModelFitMutation.SetResult, result);
         // update model params on every step
         commit(`run/${RunMutation.SetParameterValues}`, result.data.pars, { root: true });
+        commit(`run/${RunMutation.SetResult}`, state.result, { root: true });
```

Comment:
Ha, so we can get away with just re-using the whole result here because everything required in `OdinRunResult` is also in `OdinFitResult`. TypeScript isn't policing the types that go into commit parameters, but I suppose if it could it might complain about this - but we could makes the types explicitly more allowed here by making `OdinFitInputs` extend `OdinRunInputs`? 

Also, this will mean that any fit error in the result will also show up as run error, is that ok?

I was expecting this run solution to be set only on fit completion, but I suppose there's no reason not to set it on each step - this way, if you happen to change to the Run tab while fitting you'll see the Run plot updating in real time too.

Reply:
Oh sorry, I thought I had got the types right here, I can tidy that up if you'd like

Practically the error situation should be ok once the fit has started - might be worth seeing if we get caught out here.

It should be easy to knock out the model code has been compiled error message (I'll break that one apart after the fit one I am doing now).

I was originally going to do it just at the end, but this ended up feeling a bit more natural - I think performance wise it's ok though? It should only trigger a redraw when the tab is visible?